sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,560 @@
+    "    enable_online_store=True,\n",
+    "    online_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+new_kms_key['KeyMetadata']['KeyId']\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "### How to create an Offline Feature store with your KMS key? \n",
+    "Similar to the above, set `enable_online_store` to be `False` and then specify your KMS key as parameter `offline_store_kms_key_id`. You will need to substitute your Account number in `arn:aws:kms:us-east-1:123456789012:key/` replacing `123456789012` with your Account number.\n",
+    "\n",
+    "```\n",
+    "customers_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False, \n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+ new_kms_key['KeyMetadata']['KeyId']\n",
+    ")\n",
+    "\n",
+    "orders_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False,\n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+new_kms_key['KeyMetadata']['KeyId']\n",
+    ")\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this example we create an online feature store that encrypts your data using your KMS key.\n",
+    "\n",
+    "**Note**: You will need to substitute your Account number in `arn:aws:kms:us-east-1:123456789012:key/` replacing `123456789012` with your Account number."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False, \n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+ new_kms_key['KeyMetadata']['KeyId']\n",
+    ")\n",
+    "\n",
+    "orders_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False,\n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+new_kms_key['KeyMetadata']['KeyId']\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to verify that your KMS key is being used to encrypt your data in your Online or Offline Feature Store? \n",
```

Comment:
"Verify that your KMS key is being used to encrypt your data in your online or offline feature store"